### DISCUSSION TITLE
Targeting Other FPGA Boards

### DISCUSSION THREAD

**Discussion by RB-Finch** [openhwgroup/cva6#787] [Q&A]
Hi,

I am try to work out how much work is involved in targeting a different FPGA board: we have an Alveo U280. Is it a quick job to retarget to this board? Anyone have experience of this?

Many thanks,
Rob.

**Comment by RaphaelKlink** [comment]
Hey Finch,

i already had a look into it.

Unfortunantly it is not an easy retargeting. The Alveo U280 is a compute card and not a board. There it does not have any usable ports.

Therefore while the programming on the alveo card is easy, i have not found a way to connect to the core afterwards. There is no pmod or uart built into the card. There is a theoretical possbility to use the internal jtag, that is used to program the fpga, as the jtag for the core but i was not able to get it working.

I did tried it some time ago. Maybe in the mean time the pcie support for the core is now working. The alveo cards also have pcie. That would be my best guess.

Greetings Raphael

**Reply by RB-Finch** [reply]
Hi Raphael,

Thanks for the reply - very interesting. 

Do you know the best way to find out the status of the PCIe support on the core? Do you think working PCIe would definitely allow us (and you!) to use the Alveo board?

Otherwise, might be a case a buying another board then...

Many thanks,
Rob
